### PR TITLE
Burdocks added to flora, with associated recipes.

### PR DESCRIPTION
**Pull request by Slimyreg**
#### Summary

```SUMMARY: Content "Adds burdock as a wild vegetable found at the same rate as dandelions, and adds recipes when both are used in cooking."```

#### Purpose of change

Fields could always use some more colour variety and dandelions give it more life. Burdock is all over NA and New England, considered a weed, its roots can be used to make teas and drinks, together with dandelion it is also known as sarsaparilla.

#### Describe the solution

Adds burdock and dandelion burdock recipes.

#### Describe alternatives you've considered

Not doing it.

#### Testing

No testing as of yet (an update will occur when tested).

#### Additional context

Should have probably been part of a much bigger PR to include other plants, but just introducing a single weed actually proved to require a lot of JSON edits.

Most of this was done with notepad and in the firefox editor so forgive my poor alignment / formatting.

### CODE REVIEW

**Comment by Funguss** [comment]
Seeing "Notepad" immediately sets off alarm bells. It's probably fine, but notepad can screw the formatting of files entirely, beyond just alignment. Depending on the situation you can end up with files that look alright but are essentially corrupted to the point of no return. Please get yourself a good text editor!

**Comment by ashGlaw** [comment]
Cool addition! Always happy to see more local flora

**Comment by I-am-Erk** [comment]
I don't think fried/cooked burdock recipes should be autolearn, on the basis that without appropriate references, most people wouldn't be able to identify burdock on sight reliably, let alone know that it tastes nice fried.

**Comment by Funguss** [comment]
> 
> 
> I don't think fried/cooked burdock recipes should be autolearn, on the basis that without appropriate references, most people wouldn't be able to identify burdock on sight reliably, let alone know that it tastes nice fried.

I guess that depends? Burdock is a pretty obvious plant and D&B is a pretty common drink. Hell, there's even a picture of the plants on some bottles. Starting with survival 1 should learn it at least IMO. I wouldn't even call myself level 0 in real life but I know burdock is edible and how to spot it, though I guess if you went by the bottle picture you might get it confused with something.

**Comment by I-am-Erk** [comment]
The goal with autolearn is to move things into skill groups, with "survival" representing a much more vague concept. It shouldn't assume you know how to cook with and prepare plants unless cooking and preparing those plants is completely obvious. In this case, unless NE is very different from the rest of North America, dandelion and burdock drinks are quite uncommon - I've only ever seen them in specialty British stores - and someone learning survival from first principles would probably need a tip off.

**Comment by I-am-Erk** [comment]
Oh... For a future PR we may also want to add burdock root. I love me a good gobo salad.

**Comment by Slimyreg** [comment]
> 
> 
> Oh... For a future PR we may also want to add burdock root. I love me a good gobo salad.

I was thinking about it, I'll look over the survival books.

File: data/mods/Brighter_Days/modinfo [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "type": "MOD_INFO",
```
Comment (I-am-Erk) [inline data/mods/Brighter_Days/modinfo]: Pretty sure you committed this to the wrong branch

File: data/json/furniture_and_terrain/furniture-flora.json [hunk 2]
```diff
@@ -27,6 +27,21 @@
     "harvest_by_season": [ { "seasons": [ "spring", "summer", "autumn" ], "entries": [ { "drop": "raw_dandelion", "base_num": [ 1, 4 ] } ] } ],
     "bash": { "str_min": 2, "str_max": 6, "sound": "crunch.", "sound_fail": "whish." }
   },
+  {
+    "type": "furniture",
+    "id": "f_burdock",
+    "name": "burdock",
+    "description": "A common weed with a purple thistle-like flower.  It seeds tend to stick to clothing and fur.",
```
Comment (Night-Pryanik) [inline data/json/furniture_and_terrain/furniture-flora.json]:
```suggestion
    "description": "A common weed with a purple thistle-like flower.  Its seeds tend to stick to clothing and fur.",
```

File: data/json/items/comestibles/alcohol.json [hunk 3]
```diff
@@ -709,6 +709,32 @@
     "fun": 16,
     "freezing_point": 25
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "burdock_wine",
+    "name": { "str": "burdock wine", "str_pl": "burdock wine" },
+    "weight": "35 g",
```
Comment (Night-Pryanik) [inline data/json/items/comestibles/alcohol.json]: 35 grams in 250 ml? It's 7 times less denser than water.

File: data/mods/Brighter_Days/modinfo (hunk 1 above)
Comment (Slimyreg) [reply data/mods/Brighter_Days/modinfo]: > Pretty sure you committed this to the wrong branch

Yeah what's up with that? Fixing.

File: data/json/items/comestibles/alcohol.json (hunk 3 above)
Comment (Slimyreg) [reply data/json/items/comestibles/alcohol.json]: This is inherited from dandelion wine. I bet there are other examples, but I'll correct that.

Comment (Slimyreg) [reply data/json/items/comestibles/alcohol.json]: Looking for wine density, for red merlot as a baseline we have 99g for 100ml, which is alright. If we want to abstract it, we can just simplify the chemical content of our wines to be water + ethanol with a select percentage, like 12.5% ethanol, so you would expect about 97-99g per 100ml depending on ethanol density.

I'm just gonna abstract mine and say "it's 99g per 100ml for burdock wine" and leave everyone else's wines alone until I get permission to grope them.

**Pull request merged**